DigitalExcellence/dex-backend
# Bugfix/332 project icons get deleted when they shouldnt

## Description

When you update a project but leave the project icon as it is, the icon will still get deleted.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to project detail page of a project you own
2. Click edit
3. Leave the image as it is
4. Click "edit project"
5. The project should be edited successfully 

## Link to issue
#332

Closes: #332
